tailwind.config.js

```diff
@@ -109,6 +109,10 @@ module.exports = {
       ...theme('colors')
     }),
     extend: {
+      borderWidth: {
+        1: '1px',
```

Comment:
I don't think we need the 1 here. Tailwind's [default `border`](https://tailwindcss.com/docs/border-width) and `border-b` are already 1px. So anywhere we have `border-1` we can just substitute with `border`

Reply:
Good catch, thank you!